src/components/Navigation.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { BrowserRouter as Router,
+    Switch,
+    Route,
+    Link,
+    useRouteMatch,
+    useParams
+}   from "react-router-dom";
+import IndexTypes from "../containers/IndexTypes"
+import IndexCreature from "../containers/IndexCreature"
+import CreatureTypeShowpage from '../components/CreatureTypeShowpage'
+
+const Navigation = props => {
+  return (
+    <Router>
+      <div className="container">
+        <nav className="nav">
+          <ul className="menu">
+            <li>
+              <Link to="/creatures">home</Link>
+            </li>
+            <li>
+              {/* add a placeholder for this */}
```

Comment:
Not sure why this comment is in here, curly braces here could give console errors